components/dashboard/src/Login.tsx

```diff
@@ -41,52 +40,54 @@ export function Login({ gitpodService }: { gitpodService: GitpodService }) {
-                        <h1 className="text-3xl">Log in to Gitpod</h1>
-                        <h2 className="uppercase">ALWAYS READY-TO-CODE</h2>
+                    <div>
+                        <h1 className="text-7xl">Save time<br /> with Prebuilds</h1>
                     </div>
-                    <div className="flex flex-col space-y-3">
-                        {authProviders.map(ap => {
-                            return (
-                                <button key={"button" + ap.host} className="flex-none w-full rounded-none border-none p-0 inline-flex" onClick={() => openLogin(ap.host)}>
-                                    <img className="fill-current w-5 h-5 ml-4 mr-4 my-auto" src={iconForAuthProvider(ap.authProviderType)}/>
-                                    <span className="pt-2 pb-2 mr-4 text-base break-words">Continue with {ap.host}</span>
-                                </button>
-                            );
-                        })}
+                    <div className="text-gray-400 text-lg">
+                        Gitpod continuously builds your git branches like a CI server. This means no more waiting for dependencies to be downloaded and builds to finish. <a className="a-ul" href="https://www.gitpod.io/docs/prebuilds/" target="gitpod-docs">Learn more about Prebuilds</a>
+                    </div>
+                    <div>
+                        <img src="/images/terminal.svg" className="h-64 -ml-8" />
                     </div>
                 </div>
             </div>
-            <div className="flex-none mx-auto h-20 text-center">
-                <span className="text-gray-400">
-                    By signing in, you agree to our <a className="underline" target="gitpod-terms" href="https://www.gitpod.io/terms/">terms of service</a>.
-                </span>
+        </div>
+        <div id="login-section" className="flex-grow flex w-1/2">
+            <div id="login-section-column" className="flex-grow max-w-2xl flex flex-col h-100">
+                <div className="flex-grow h-100 flex flex-row items-center justify-center" >
+                    <div className="rounded-xl px-10 py-10 mx-auto">
+                        <div className="mx-auto pb-8">
+                            <img src="/gitpod.svg" className="h-16 mx-auto" />
+                        </div>
+                        <div className="mx-auto text-center pb-8 space-y-2">
+                            <h1 className="text-3xl">Log in to Gitpod</h1>
+                            <h2 className="uppercase text-sm">ALWAYS READY-TO-CODE</h2>
+                        </div>
+                        <div className="flex flex-col space-y-3">
+                            {authProviders.map(ap => {
+                                return (
+                                    <button key={"button" + ap.host} className="btn-login flex-none w-full p-0 inline-flex" onClick={() => openLogin(ap.host)}>
```

Comment:
**nitpick**: Ideally, the width of both buttons should remain the same at all times and using a provider (host) with a long domain name could increase the width. We could truncate the button label or additionally use a tooltip to show the full domain. Minor and can be a follow up issue.